scripts/machine-launch-script.sh

```diff
@@ -1,83 +1,264 @@
 #!/bin/bash
 
+CONDA_INSTALL_PREFIX=/opt/conda
+CONDA_INSTALLER=https://github.com/conda-forge/miniforge/releases/download/4.11.0-4/Miniforge3-4.11.0-4-Linux-x86_64.sh
+CONDA_ENV_NAME="base"
+
+DRY_RUN_OPTION=""
+DRY_RUN_ECHO=()
+
+usage()
+{
+    echo "Usage: $0 [options]"
+    echo
+    echo "Options:"
+    echo "[--help]                  List this help"
+    echo "[--prefix <path>]         Install prefix for conda. Defaults to /opt/conda."
+    echo "                          If <path>/bin/conda already exists, it will be used."
+    echo "[--env <name>]            Name of environment to create for conda. Defaults to installing in base env."
+    echo "[--dry-run]               Pass-through to all conda commands and only print other commands."
+    echo "                          NOTE: --dry-run will still install conda to --prefix"
+    echo
+    echo "Examples:"
+    echo "  % $0"
+    echo "     Install into default system-wide prefix (using sudo if needed) and add install to system-wide /etc/profile.d"
+    echo "  % $0 --prefix ~/conda --env my_custom_env"
+    echo "     Install into $HOME/conda and add install to ~/.bashrc"
+    echo "  % $0 --prefix \$CONDA_PREFIX --env my_custom_env"
+    echo "     Create my_custom_env in existing CONDA_PREFIX conda install"
+    echo "     NOTE: CONDA_PREFIX is set in your environment when you activate a conda env"
```

Comment:
I forgot that conda makes the CONDA_PREFIX environment variable what is called `active_prefix` inside the tool.  These instructions need to be updated to say:
```suggestion
    echo "  % $0 --prefix \$(conda info --json | jq -r '.conda_prefix') --env my_custom_env"
    echo "     Create my_custom_env in pre-existing conda install that you are using"
```

Reply:
A less convoluted suggestion or one that at least doesn't involve a pipeline of JSON being wrangled by jq might be:
```suggestion
    echo "  % $0 --prefix \${CONDA_EXE%/bin/conda} --env my_custom_env"
    echo "     Create my_custom_env in existing conda install"
    echo "     NOTE: CONDA_EXE is set in your environment when you activate a conda env"
```